sites/about.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>About : Hyunji</title>
+    <link href ="../styles/normalize.css" rel="stylesheet">
+    <link href ="../styles/styles.css" rel="stylesheet">
+  </head>
+  <body id="about">
+    <main>
```

Comment:
Really minor comment: A navigation bar at the top of each page would help me go back and forth between your website pages more easily.  :)